packages/account/src/Components/form-body/form-body.tsx

```diff
@@ -1,5 +1,5 @@
 // [TODO] - To be removed once CFD is configured to use the new form-body component
-import React, { PropsWithChildren, Fragment } from 'react';
+import { PropsWithChildren, Fragment } from 'react';
```

Comment:
Please do not remove React. change import statement to `import React from 'react'` or keep it as it is

Reply:
This is fixed in separate PR.